modules/thermal_hydraulics/src/userobjects/ADShaftConnectedPump1PhaseUserObject.C

```diff
@@ -186,7 +186,7 @@ ADShaftConnectedPump1PhaseUserObject::computeFluxesAndResiduals(const unsigned i
 
     // a positive head value results in a positive S_momentum
     const ADRealVectorValue S_momentum = (_rhoA[0] / _A[0]) * _g * _pump_head * _A_ref * _di_out;
-    //
+    // SEEMS WRONG
```

Comment:
not sure I remember why

Reply:
Probably should remove if you can't remember why.